packages/abi-coder/src/encoding/coders/OptionCoder.test.ts

```diff
@@ -7,7 +7,7 @@ import { VoidCoder } from './VoidCoder';
  * @group browser
  */
 describe('OptionCoder', () => {
-  const coder = new OptionCoder('Option', {
+  const coder = new OptionCoder('std::option::Option', {
```

Comment:
Is `std::option::Option` representing the name or type here. I've always found the combination of both name and type for the type coder classes a bit confusing, we should probably settle on one. Probably a point for #1795.

Reply:
Not sure I follow your point, but for our context, I guess it's both because callpaths are now enabled by default.

Good to have as a checkpoint on #1795, as you said.

Reply:
After working with this `name` parameter in #2826 I realized that it does nothing and can be deleted.